notebooks/summer-school/2021/lab1.ipynb

```diff
@@ -10,6 +10,8 @@
     "    <iframe src=\"https://www.youtube.com/embed/uI02dn7PsHI\" title=\"YouTube video player\" frameborder=\"0\" allow=\"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>\n",
     "</div>\n",
     "\n",
+    "- Download the notebook: <a href=\"/content/summer-school/2021/resources/lab-notebooks/lab-1.ipynb\">[en]</a> <a href=\"/content/summer-school/2021/resources/lecture-notes/lab-1-ja.ipynb\">[ja]</a>\n",
```

Comment:
the URL for this Japanese notebook (`/content/summer-school/2021/resources/lecture-notes/lab-1-ja.ipynb`) appears to be incorrect. it believe it should be `/content/summer-school/2021/resources/lab-notebooks/lab-1-ja.ipynb`.
notice `lecture-notes` vs `lab-notebooks`